app/views/template_inputs/_invocation_form.html.erb

```diff
@@ -1,10 +1,10 @@
 <%= input_values_fields.fields_for input.id.to_s, composer.input_value_for(input) do |input_fields| %>
   <% if input.options.present? %>
-    <%= selectable_f input_fields, :value, input.options_array, { :include_blank => !input.required }, :label_help => input.description, :require => input.required, :label => input.name, :id => input.name, :onchange => local_assigns[:onchange] %>
+    <%= selectable_f input_fields, :value, input.options_array, { :include_blank => !input.required, :selected => input.default }, :label_help => input.description, :require => input.required, :label => input.name, :id => input.name, :onchange => local_assigns[:onchange] %>
```

Comment:
I am afraid `selected: <value>` will override the value, so it will select the default value even on editing.

Reply:
I've checked and it works as it should. When you change selected default to prefered value it picks new selected (thus overrides the selected by default value).

Have you tried it or it is just a guess?

Reply:
It's how native rails `f.select` works, but I haven't tested it, I'm going to do that and let you know :)

Reply:
I have tested an it really behaves as I were expecting, if I remove the `selected: ...` it behaves as expected though.